app/src/main/kotlin/com/wire/android/notification/NotificationChannelsManager.kt

```diff
@@ -48,14 +48,11 @@ class NotificationChannelsManager @Inject constructor(
         )
     }
 
-    fun createNotificationChannels(allUsers: List<SelfUser>) {
-        appLogger.i("${TAG}: creating all the channels for ${allUsers.size} users")
+    // Creating user-specific NotificationChannels for each user, they will be grouped by User in App Settings.
+    fun createUserNotificationChannels(allUsers: List<SelfUser>) {
+        appLogger.i("${TAG}: creating all the notification channels for ${allUsers.size} users")
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) return
 
-        // creating regular NotificationChannels, that are common for all users and shouldn't be grouped.
-        createRegularChannel(NotificationConstants.WEB_SOCKET_CHANNEL_ID, NotificationConstants.WEB_SOCKET_CHANNEL_NAME)
```

Comment:
Why was the common channel for all users removed? 🤔 Or was just moved to `PersistentWebSocketService.kt` in `generateForegroundNotification()`?

Reply:
Exactly, it was moved directly to the service, all other "global" channels were moved earlier, this one was left